automation_tools/satellite6/upgrade/template.py

```diff
@@ -0,0 +1,121 @@
+from automation_tools.satellite6.hammer import hammer, set_hammer_config
+from automation_tools.satellite6.upgrade import \
+    check_necessary_env_variables_for_upgrade
+from automation_tools.satellite6.upgrade.tasks import \
+    get_rhevm_client, wait_till_rhev_instance_status
+from fabric.api import run, execute
+from ovirtsdk.xml import params
+from time import sleep
+import os
+import sys
+
+
+# Restart the Services
+def service_restart():
+    try:
+        services = run('katello-service restart')
+        if services.return_code > 0:
+            sys.exit(1)
+    except Exception as ex:
```

Comment:
The try/except block is not needed. If the exception is not catch then it will sys.exit and will print the exception information.